w3d_translator/threads.py

```diff
@@ -0,0 +1,68 @@
+
+import os
+import threading
+import typer
+
+# from unity import create_project, copy_files, add_empty_scene
+from xml_to_unity import xml_to_unity
+
+
+# Thread class for translating a single project
+class projectThread (threading.Thread):
```

Comment:
classes are typically `TitleCase`
```suggestion
class ProjectThread (threading.Thread):
```